internal/urls.py

```diff
@@ -19,7 +19,7 @@
     path("members/<int:pk>/retire", permission_required("internal.can_edit_group_membership")(MemberRetireView.as_view()), name="member-retire"),
     path("members/<int:pk>/retire/undo", permission_required("internal.can_edit_group_membership")(MemberUndoRetireView.as_view()), name="member-undo-retire"),
     path("members/access/<int:pk>/change", ToggleSystemAccessView.as_view(), name="toggle-system-access"),
-    # TODO: Change to the HomeView when there is actually some content to show there
+    path("", Home.as_view(), name="home"),
     path("", MembersListView.as_view(), name="home"),
```

Comment:
I didn't notice before now, but there are now two views with the same name, `home` 😅 Following the instructions of the removed comment, the following should be done:
```suggestion
    path("", HomeView.as_view(), name="home"),
```

`Home` should also be renamed to `HomeView`, as I've done above, to make it clearer that it's a view and not, say, a model 😛 (I know the view was created a long time ago, but it would be nice if the name could be improved as part of this PR 🙂)

Reply:
Definitely , thank you. I must have missed the duplication in the merge process.